src/lib/utils/thorchain/hooks/useSendThorTx.tsx

```diff
@@ -53,63 +51,58 @@ type Action =
   | 'withdrawSavers'
 
 type UseSendThorTxProps = {
-  amountCryptoBaseUnit: string | undefined
-  assetId: AssetId | null
-  memo: string | undefined
   accountId: AccountId | null
+  action: Action
+  amountCryptoBaseUnit: string | undefined
+  assetId: AssetId
+  disableRefetch?: boolean
```

Comment:
More declarative, and I think we should never use `isSubmitting` for anything else so might as well  💜